frontend/src/components/pages/GlobalForms/index.tsx

```diff
@@ -194,6 +194,45 @@ const GlobalFormsPage = (): React.ReactElement => {
     }
   };
 
+  const onDeleteCustomQuestion = (questionToBeDeleted: CreateFormQuestion) => {
+    setFormTemplateQuestions((oldArr: CreateFormQuestion[]) =>
+      oldArr.filter(
+        (question: CreateFormQuestion) => question !== questionToBeDeleted,
+      ),
+    );
+    toast({
+      description: "Question has been successfully deleted.",
+      status: "success",
+      duration: 3000,
+      isClosable: false,
+      variant: "subtle",
+    });
+  };
+
+  const onEditCustomQuestion = (
+    oldQuestion: CreateFormQuestion,
+    newQuestion: CreateFormQuestion,
+  ) => {
+    setFormTemplateQuestions((oldArr: CreateFormQuestion[]) => {
+      const newArr = [...oldArr];
+      for (let i = 0; i < newArr.length; i += 1) {
+        if (newArr[i] === oldQuestion) {
+          newArr[i] = newQuestion;
+          break;
+        }
+      }
+      return newArr;
+    });
+
```

Comment:
I think you missed the API call to the backend to edit and delete the form questions from the template. 
See the following endpoints:
1. remove question from form template `DELETE "/formTemplate/formQuestion/:formQuestionId"`
2. edit question in template `PATCH "/formTemplate/formQuestion/:oldQuestionId"`

Reply:
Added now